deployment/migrations/versions/0033_1c06d0ade60c_calculate_costs_statically.py

```diff
@@ -30,48 +30,6 @@
 depends_on = None
 
 
-hardcoded_initial_price: Dict[ProductPriceType, ProductPricing] = {
-    ProductPriceType.PROGRAM: ProductPricing(
-        ProductPriceType.PROGRAM,
-        ProductPrice(
-            ProductPriceOptions("0.05", "0.000000977"),
-            ProductPriceOptions("200", "0.011")
-        ),
-        ProductComputeUnit(1, 2048, 2048)
-    ),
-    ProductPriceType.PROGRAM_PERSISTENT: ProductPricing(
-        ProductPriceType.PROGRAM_PERSISTENT,
-        ProductPrice(
-            ProductPriceOptions("0.05", "0.000000977"),
-            ProductPriceOptions("1000",  "0.055")
-        ),
-        ProductComputeUnit(1, 20480, 2048)
-    ),
-    ProductPriceType.INSTANCE: ProductPricing(
-        ProductPriceType.INSTANCE,
-        ProductPrice(
-            ProductPriceOptions("0.05", "0.000000977"),
-            ProductPriceOptions("1000", "0.055")
-        ),
-        ProductComputeUnit(1, 20480, 2048)
-    ),
-    ProductPriceType.INSTANCE_CONFIDENTIAL: ProductPricing(
-        ProductPriceType.INSTANCE_CONFIDENTIAL,
-        ProductPrice(
-            ProductPriceOptions("0.05", "0.000000977"),
-            ProductPriceOptions("2000", "0.11")
-        ),
-        ProductComputeUnit(1, 20480, 2048)
-    ),
-    ProductPriceType.STORAGE: ProductPricing(
-        ProductPriceType.STORAGE,
-        ProductPrice(
-            ProductPriceOptions("0.333333333"),
-        )
-    ),
-}
-
-
```

Comment:
I think that we shouldn't have to modify a migration already executed on the rest of the CCNs to prevent unconsistencies on the database.